test/test_detection.py

```diff
@@ -13,12 +13,16 @@
 from .conftest import DATAPATH
 
 
-lofarcasatable = os.path.join(DATAPATH, 'casatable/L55596_000TO009_skymodellsc_wmax6000_noise_mult10_cell40_npix512_wplanes215.img.restored.corr')
-casatable = os.path.join(DATAPATH, 'aartfaac.table')
-fitsfile = os.path.join(DATAPATH, 'lofar.fits')
-hdf5file = os.path.join(DATAPATH, 'lofar.h5')
+lofarcasatable = os.path.join(DATAPATH, ('casatable/L55596_000TO009_' +
+                                         'skymodellsc_wmax6000_noise_mult10' +
+                                         '_cell40_npix512_wplanes215.img' +
+                                         '.restored.corr'))
```

Comment:
I'm not a big fan of breaking long strings like this.  It reduces searchability. I guess you are trying to reduce line lengths, we can start using `black`.  Any kind of human controlled formatting is a no-no.

If we decide to use black, we have to plan it properly, otherwise it'll break diffs.  The same comment applies to the other one in `test_accessors.py`

Reply:
Yes, I see your point, it indeed reduces searchability. My linter complained about this line exceeding 120 characters, that is why I adjusted it, but I did not think of searchability.

Pls. let me decide on using `black` later.
I am fine with any temp solution for now, since I consider this a minor issue.

Reply:
Let's leave it as is then, we can fix it later.